content/overview/pricing/managed-services-cost.md

```diff
@@ -9,6 +9,8 @@ aliases:
   - /overview/pricing/other-services/
 ---
 
-Currently, we don’t charge for [managed services]({{< relref "overview/pricing/pricing-terminology.md#managed-services" >}}). We plan to start charging for them in the future to account for the resources they consume and our work to provide them. The pricing for each service will be different, and they will be severable. We will ensure that users of these services are aware of the rates when they are set, and they will not be charged before then.
+Currently, we don’t charge individually for [managed services]({{< relref "docs/apps/managed-services.md" >}}) (services that cloud.gov can spin up quickly and run on your behalf, such as databases, storage, and caching).
```

Comment:
'that cloud.gov can spin up quickly...'  can we change this to ensure that the self-service nature is reflected. Otherwise it sounds like these are services the customer must ask cloud.gov to run.

Reply:
Updated!